Pokedex_ios.xcodeproj/xcshareddata/xcschemes/Pokedex_iosTests.xcscheme

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does the unit tests run on your local? with this changes circle ci is not able to run this tests

![imagen](https://user-images.githubusercontent.com/1596425/179008022-a5d150a0-ed9e-4cca-b358-f6818c9e5374.png)